index.js

```diff
@@ -1,13 +1,39 @@
 class SortedList {
-  constructor() {}
+  
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+     this.length++;
+   return this.items.sort(function(a, b){
+   return a - b;
+  })
+   return this.items;
```

Comment:
this `return` statement will never be called because you have one before this line in the same function